scipy/spatial/qhull.pyx

```diff
@@ -1847,6 +1847,14 @@ class Delaunay(_QhullUser):
     >>> tri.find_simplex(p)
     array([ 1, -1, 1], dtype=int32)
 
+    The returned integers in the array are the indices of the simplex the 
+    corresponding point is in. If -1 is returned, the point is in no simplex.
+    
+    >>> p = np.array([(0.1, 0.2), (0.5, 1.05)])
```

Comment:
The test suite failed (https://travis-ci.org/scipy/scipy/jobs/245380893) because you're changing ``p`` here and it's used again afterwards. Can you just rename ``p`` to ``p_valids`` or something else?